pkg/utils/unit.go

```diff
@@ -8,3 +17,33 @@ const (
 	GB
 	TB
 )
+
+type logCollector struct {
```

Comment:
Could you move it to a new package `pkg/summary`?

Reply:
sure, [moved](https://github.com/pingcap/br/pull/118/files#diff-500e3c8287bac6a5d41b94b101065518R13)